service/lambdas/employer/add_employer_job.py

```diff
@@ -0,0 +1,48 @@
+from http import HTTPStatus
+
+from aws_lambda_context import LambdaContext
+from pydantic import ValidationError
+from aws_lambda_powertools import Logger
+from service.models.employer.employer_job import EmployerJob
+from service.models.employer.employer import Employer
+from service.common.utils import get_env_or_raise
+from service.lambdas.employer.constants import EmployerConstants
+import uuid
+import boto3
+from datetime import datetime
+from decimal import Decimal
+
+logger = Logger()
+
+
+# POST /jobli/employers/{employer_id}/jobs
+@logger.inject_lambda_context(log_event=True)
+def add_employer_job(event: dict, context: LambdaContext) -> dict:
+    try:
+        if 'pathParameters' not in event or not event['pathParameters'] \
+                or 'employer_id' not in event['pathParameters'] or 'body' not in event or not event['body']:
+            return {'statusCode': HTTPStatus.BAD_REQUEST,
+                    'headers': {'Content-Type': 'application/json'},
+                    'body': "Missing employer job body/path params to create"}
+        employer_job: EmployerJob = EmployerJob.parse_raw(event['body'])
+        employer_job.job_id = str(uuid.uuid4())
+        employer_job.employer_id = event["pathParameters"]["employer_id"]
+        employer_job.created_time = Decimal(datetime.now().timestamp())
+        dynamo_resource = boto3.resource("dynamodb")
+        employers_table = dynamo_resource.Table(get_env_or_raise(EmployerConstants.EMPLOYERS_TABLE_NAME))
+        jobs_table = dynamo_resource.Table(get_env_or_raise(EmployerConstants.JOBS_TABLE_NAME))
+        # Check if employer id exists, will throw exception if not
+        Employer.parse_obj(employers_table.get_item(
+            Key={"employer_id": employer_job.employer_id})['Item'])
```

Comment:
if there's no employer ID, then there isn't going to be an 'Item' key in the return value, so it will throw KeyError, need to catch that and return a BAD_REQUEST (right now it'll be caught by "except Exception" and return INTERNAL_SERVER_ERROR)